integrasjonspunkt/src/test/kotlin/no/difi/meldingsutveksling/ks/fiksio/FiksIoServiceTest.kt

```diff
@@ -0,0 +1,83 @@
+package no.difi.meldingsutveksling.ks.fiksio
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import no.difi.meldingsutveksling.ServiceIdentifier
+import no.difi.meldingsutveksling.api.OptionalCryptoMessagePersister
+import no.difi.meldingsutveksling.domain.sbdh.StandardBusinessDocument
+import no.difi.meldingsutveksling.nextmove.NextMoveOutMessage
+import no.difi.meldingsutveksling.pipes.Plumber
+import no.difi.meldingsutveksling.pipes.PromiseMaker
+import no.difi.meldingsutveksling.serviceregistry.ServiceRegistryLookup
+import no.difi.meldingsutveksling.serviceregistry.externalmodel.ServiceRecord
+import no.ks.fiks.io.client.FiksIOKlient
+import no.ks.fiks.io.client.model.*
+import org.junit.Before
+import org.junit.Test
+import java.util.*
+import kotlin.test.assertEquals
+
+internal class FiksIoServiceTest {
+
+    @MockK
+    lateinit var fiksIOKlient: FiksIOKlient
+
+    @MockK
+    lateinit var serviceRegistryLookup: ServiceRegistryLookup
+
+    @MockK
+    lateinit var persister: OptionalCryptoMessagePersister
+
+    @MockK
+    lateinit var objectMapper: ObjectMapper
+
+    @MockK
+    lateinit var plumber: Plumber
+
+    @MockK
+    lateinit var promiseMaker: PromiseMaker
+
+    private lateinit var fiksIoService: FiksIoService
+
+    @Before
+    fun before() {
+        MockKAnnotations.init(this)
+        fiksIoService = FiksIoService(fiksIOKlient, serviceRegistryLookup, persister, objectMapper, plumber, promiseMaker)
+    }
+
+    @Test
```

Comment:
Dekkar denne eine testen all funksjonalitet? ser den opprettar request og sender melding, skal mottak testast?

Reply:
Lagt til test for mottak.